testing/integration/testdata/.upgrade-test-agent-versions.yml

```diff
@@ -0,0 +1,12 @@
+# This file is generated automatically. Please do not manually edit it.
```

Comment:
We do manually edit this occasionally when we need to temporarily block a version because of a bug, but in general that is the exception.